Misc/tools/locust_load_test/README.md

```diff
@@ -0,0 +1,33 @@
+# LocustIO Load Test
```

Comment:
This README could be more specific about what UI means (client? server? both?)

Reply:
I am really just hitting the server endpoints. So I will just change it to UI-Server